Defs/ThingDefs_Buildings/Buildings_Industry.xml

```diff
@@ -191,7 +191,7 @@
   <!--==================== Tribute To Tornsworth ====================-->
   <ThingDef ParentName="PRF_IndustrialBase">
     <defName>PRF_MrTsArtMachine</defName>
-    <label>Thornsworth's art machine</label>
+    <label>Thornsworth art machine</label>

```

Comment:
Would not removed `'s` here.

Reply:
Ok, but why "Thornsworth holographic storage" but not "Thornsworth art machine"?